recognition/src/ransac_based/orr_octree_zprojection.cpp

```diff
@@ -99,23 +100,24 @@ pcl::recognition::ORROctreeZProjection::build (const ORROctree& input, float eps
   // Compute the bounding box of the full leaves
   const vector<ORROctree::Node*>& full_leaves = input.getFullLeaves ();
   vector<ORROctree::Node*>::const_iterator fl_it = full_leaves.begin ();
-  float full_leaves_bounds[4];
+  std::array<float, 4> full_leaves_bounds;
 
   if ( full_leaves.empty() )
     return;
 
   // The initialization run
-  full_leaves_bounds[0] = (*fl_it)->getBounds ()[0];
-  full_leaves_bounds[1] = (*fl_it)->getBounds ()[1];
-  full_leaves_bounds[2] = (*fl_it)->getBounds ()[2];
-  full_leaves_bounds[3] = (*fl_it)->getBounds ()[3];
```

Comment:
Were these lines filling the bounds with uninitialized trash?

Reply:
They don't.
Old variant (current master): Fill `full_leaves_bounds` at start with first item of `full_leaves` and then start loop with **second** item.
New variant (this PR): Fill `full_leaves_bounds` at start with values which will be always overridden in first loop iteration and then start loop with **first** item.